src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,42 @@
+<a href="/board/{{board._id}}">Back</a>
+<div id="boardSettings">
+    <!-- Form for changing the board settings -->
+    <form action="/board/settings/{{board._id}}" method="POST">
+        <div class="boardNameDiv">
+            <label for="boardName">Name: </label>
+            <input id="boardName" type="text" placeholder="Board Name" value="{{board.boardName}}"/>
+        </div>
+        <div class="colorDiv">
+            <label for="positionValue">Color: </label>
+            <input id="positionValue" type="text" placeholder="Color" value="{{board.color}}"/>
+        </div>
+        <div class="descriptionDiv">
+            <label for="positionValue">Description: </label>
+            <input id="positionValue" type="text" placeholder="Description" value="{{board.description}}"/>
+        </div>
+        <div class="submitButtonDiv">
+            <button type="submit">Submit</button>
+        </div>
+    </form>
+    <!-- form for deleting a board -->
+    <form action="/board" method="POST">
+        <div class="deleteButtonDiv">
+            <button type="submit">Delete Board</button>
```

Comment:
I like what Marco mentioned here, where this should say 'Delete Board' when you are the only member on the board, and 'Leave Board' when thats not the case. I can give you additional information from the route to handle this
Let's say you now have access to a variable {{onlyMember}} which is true if you are the only person on the board.

Reply:
To accomplish this, go to src/routes/board.js
and on line 1075 add:
const onlyMember = (boardInfo.members === 1);
and change the res.render to also have onlyMember: onlyMember:
res.render('board-settings', { title: 'Board Settings', board: renderInfo, onlyMember: onlyMember });

Reply:
Are these actions both going to the same route? and if not, specifically where is the leave board form posting to?

Reply:
same route, the backend handles the logic of deleting the board if only one person remains